s-pipes-modules-utils/s-pipes-module-archetype/bin/generate-test-module.sh

```diff
@@ -5,15 +5,16 @@
 # Change the variables below as necessary
 ARCHETYPE_GROUP_ID="cz.cvut.kbss"
 ARCHETYPE_ARTIFACT_ID="s-pipes-module-archetype"
-ARCHETYPE_VERSION="0.4.0"
+ARCHETYPE_VERSION="1.0-SNAPSHOT"
```

Comment:
```suggestion
ARCHETYPE_VERSION="0.4.0"
```

@Matthew-Kulich I  suggest updating this module version in sync with other modules.

In this way we would inhere 0.4.0 from s-pipes-parent. See my last  two commits.

Reply:
Agree. I saw them, it looks good